arrows/qt/EmbeddedPipelineWorker.cxx

```diff
@@ -157,6 +159,7 @@ ::processOutput(
 
   if( output->is_end_of_data() )
   {
+    this->atEnd = true;
```

Comment:
Do we ever reuse a pipeline worker after it has processed a video to completion?  I don't see anywhere that `atEnd` is set to `false` except in the constructor.  Should we set this back to `false` when you call `run()`?

Reply:
> Do we ever reuse a pipeline worker after it has processed a video to completion?

Not to my knowledge. However, it looks like it may otherwise be possible to do so, so I made the change. Thanks!

Reply:
@mleotta, can you please re-review? Thanks!